hw2/03-count.js

```diff
@@ -1,3 +1,24 @@
-// Add your code here
+const input = document.querySelector("input");
+const result = document.querySelector("#results");
 
-input.addEventListener('keydown', handleKeyDown);
+let userInput = "";
+
+function handleKeyDown(event) {
+    userInput = (event.target.value + event.key).toLowerCase();
+    updateResults();
+}
+
+function updateResults() {
+    const text = result.textContent.toLowerCase();
```

Comment:
Searching replaces every word with its lowercase, which should not be the case. It should only highlight words.

Reply:
This issue has been fixed.  I didnt know it was actually changing the word to lowercase.  Thanks!